docs/iceberg-rest-service.md

```diff
@@ -18,10 +18,7 @@ The Apache Gravitino Iceberg REST Server follows the [Apache Iceberg REST API sp
   - multi table transaction
   - pagination
 - Works as a catalog proxy, supporting `Hive` and `JDBC` as catalog backend.
-- Supports multi storage.
-  - HDFS
-  - S3
-  - OSS
+- Supports diverse storage like `S3`, `HDFS`, `OSS`, and provides the capability to support other storages.
```

Comment:
I believe there' gcs, right? Please list all the supported cloud storage.

Reply:
"Supports different cloud storages...: "

Reply:
using `Supports different storages`,  because HDFS is not cloud storage